src/NuGet.Core/NuGet.ProjectModel/TargetFrameworkInformation.cs

```diff
@@ -19,21 +19,37 @@ public class TargetFrameworkInformation : IEquatable<TargetFrameworkInformation>
         /// A fallback PCL framework to use when no compatible items
         /// were found for <see cref="FrameworkName"/>. This check is done
         /// per asset type.
+        /// Imports will contain either ATF or PTF, whichever one is being used.
         /// </summary>
-        /// <remarks>Deprecated. Use <see cref="AssetTargetFallback" /> instead.</remarks>
-        /// <see cref="Imports" /> cannot be used with <see cref="AssetTargetFallback" />.
+        /// <remarks>Deprecated. Use <see cref="AssetTargetFallbacks" /> instead.</remarks>
+        /// <see cref="Imports" /> cannot be used with <see cref="AssetTargetFallbacks" />.
         public IList<NuGetFramework> Imports { get; set; } = new List<NuGetFramework>();
 
         /// <summary>
+        /// This is just kept for legacy readers.
+        /// </summary>
+        public bool AssetTargetFallback { get; set; }
```

Comment:
a bool should usually start with Has or Is or something like that which clearly indicates it is a bool.